client/state/comments/actions.js

```diff
@@ -56,21 +56,20 @@ export const requestCommentsList = query => ( {
 	query,
 } );
 
-/***
- * Creates a remove comment action for a siteId, postId, commentId
+/**
+ * Creates an action that permanently deletes a comment
```

Comment:
Yup, I actually tried right now. :)
But then again, there's nothing that checks if it's `trash` before deleting.
Shouldn't be hard to add it and dispatch `changeCommentStatus` instead (all the data is already there).